ParticleSetupResources/setup.storyboard

```diff
@@ -27,7 +25,7 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="hPd-D1-g0F">
-                                <rect key="frame" x="0.0" y="64" width="320" height="504"/>
+                                <rect key="frame" x="0.0" y="44" width="320" height="524"/>
```

Comment:
Are the storyboard changes expected?

Reply:
This happens every time you open the storyboard. Most of the times I reset them before committing, but if I know I have changed something in the storyboard, I usually commit it.